src/View/Helper/FormHelper.php

```diff
@@ -674,6 +674,9 @@ public function isFieldError($field)
      */
     public function error($field, $text = null, array $options = [])
     {
+        if (substr($field, -5) === '._ids') {
+            $field = substr($field, 0, -5);
+        }
```

Comment:
It might have, say for example you wanted to validate that a maximum/minimum of x tags is being selected.